src/components/displayName-form-item.yml

```diff
@@ -0,0 +1,9 @@
+enterDisplayName:
+  en: Please enter a display name
+  de: Bitte geben Sie einen Anzeigenamen an
+displayNameIsTooShort:
+  en: The display name must be at least {length} characters long
+  de: Der Anzeigenamen muss mindestens {length} Zeichen lang sein
+displayNameIsTooLong:
+  en: The display name must be at most {length} characters long
+  de: Der Anzeigenamen muss höchstens {length} Zeichen lang sein
```

Comment:
Will change these 2 in Nominativ case as well.

Reply:
Actually, it should be "Der Anzeigename **darf** höchstens {length} Zeichen lang sein" (as oppoesed to "Der Anzeigenamen **muss** mindestens {length} Zeichen lang sein")

Reply:
Thanks for the correction and side-explanation! Updated